private/rules/java_export.bzl

```diff
@@ -91,6 +91,7 @@ def java_export(
     javadocopts = kwargs.pop("javadocopts", [])
     doc_deps = kwargs.pop("doc_deps", [])
     doc_url = kwargs.pop("doc_url", "")
+    doc_resources = kwargs.pop("doc_resources", [])
```

Comment:
why are these kwargs and not args on the rule as attrs

Reply:
I just followed the pattern of the other doc arguments, I'm not sure the exact reason.